### ISSUE TITLE
Remove stop reasons from events

### ISSUE THREAD

**Issue by autodidaddict** [wasmCloud/wasmCloud#6]
This is a philosophical decision, possibly "opinionated". Right now when actors and providers stop, they publish events indicating such. The problem is they're currently required to figure out _why_ they've stopped and supply some kind of enum to describe that. Years of running distributed systems tells me that this is not meaningful information and, more often than not, the reason why will be incorrect anyway, because the actors shouldn't have to guess why they've been shut down... 

In short - stop requiring stop reasons on events emitted through the control plane, which means removing those reasons from the `ControlEvent` enum and then cleaning up all the consuming code from the various publication points.

**Comment by mikezupper** [comment]
I created a PR to address this issue. Please take a look

**Comment by autodidaddict** [comment]
This has been resolved. In the future I should be more diligent about linking issues to PRs.
